### ISSUE TITLE
add logger attribute to Model class

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#133]
can pass a logger to Model upon instantation,
if logger is not None during train / eval, 
will log statements to stdout  / file, e.g. validation accuracy between epochs.

Will add SummaryWriter too, but want to capture what's getting sent to stdout
